dinstaller-cli/src/main.rs

```diff
@@ -24,6 +24,21 @@ struct Cli {
 fn probe() {
     let client = manager::ManagerClient::new(dinstaller_lib::connection().unwrap()).unwrap();
     client.probe().unwrap();
+    show_progress(client)
+}
+
+fn install() {
+    let client = manager::ManagerClient::new(dinstaller_lib::connection().unwrap()).unwrap();
```

Comment:
We can import the `connection` function if preferred.

Reply:
so far not critical. Lets see how other calls will look like.